@xen-orchestra/web-core/lib/packages/job/README.md

```diff
@@ -0,0 +1,130 @@
+# Job System Documentation
+
+The Job System provides a type-safe way to manage asynchronous operations with built-in state handling and identity-based concurrency control.
+
+## Core Concepts
+
+- **Job**: An asynchronous operation with validation, state management, and concurrency control
+- **Job Arguments**: Typed parameters that can be single values or arrays
+- **Job Identity**: Values used to track concurrent executions of the same job
+
+## Usage
+
+### When to use argument identity?
+
+You'll want to use a Job argument as identity when two different values means two different processes.
+
+Let's imagine a job to save a document, which will take `id` and `content` as arguments.
+
+Should the job be tracked by the document ID? Yes, because `isRunning` could be `true` for file A and `false` for file B.
+
+Should the job be tracked by the document content? No, because `isRunning` should stay `true` for a specific file, even it the `content` value changes.
```

Comment:
```suggestion
Should the job be tracked by the document content? No, because `isRunning` should stay `true` for a specific file, even if the `content` value changes.
```